### DISCUSSION TITLE
The SAHI tool can be utilized for the purpose of image segmentation.

### DISCUSSION THREAD

**Discussion by shznky** [obss/sahi#952] [Q&A] [answered]
I want to use SAHI for segmentation tasks in yolov8, how does this work?

**Comment by fcakyon** [comment, marked as answer]
Segmentation mask support for yolov8, mmdetection, detectron2, torchvision models is live with the latest release! @shznky   https://github.com/obss/sahi/discussions/1051

There is also 5x memory and speed improvement when working with masks!

Check the demo notebook on how it works: https://github.com/obss/sahi/blob/main/demo/inference_for_yolov8.ipynb
